### DISCUSSION TITLE
How to get the user in the webmail on login ?

### DISCUSSION THREAD

**Discussion by zenixLR** [modoboa/modoboa#3248] [Help] [answered]
Hello,
Every times a simple user log-in, his first page is the Profile settings.
What is the simplest way to make him start directly in the webmail instead?
Thanks you for your answer.

**Comment by tonioo** [comment, marked as answer]
There is a setting for that. Go to online parameters and you should find something called "Default top redirection"

**Reply by zenixLR** [reply]
Found!
Thanks You.

Modoboa is great, easy install but free to customize, I like it a lot.
